File: Adi işlər/Riyazi hesablamalar/fibonacci.py
def fibonacci(n):
    if n <= 1:
        return n
    else:
        return fibonacci(n-1) + fibonacci(n-2)

# İstifadəçidən neçəinci ədədi istədiyini alırıq
n = int(input("Fibonacci ardıcıllığının neçənci ədədini istəyirsən? "))

print(f"{n}-ci Fibonacci ədədi: {fibonacci(n)}")

File: Adi işlər/Riyazi hesablamalar/Fibonacci.js
var n = parseInt(prompt("Fibonacci ardıcıllığının neçənci üzvünü görmək istəyirsiniz?"))

function fibonacci_rekursiv(n){
    if  (n <= 0){
        print("0")
    } else if (n == 1) {
        print("1")
    } else {
        return fibonacci_rekursiv(n-1) + fibonacci_rekursiv(n-2)
}}

